crates/red_knot/tests/cli.rs

```diff
@@ -38,11 +37,7 @@ fn test_respect_ignore_files() -> anyhow::Result<()> {
     ");
 
     // Test that we can set to false via config file
-    let case = TestCase::with_files([
-        ("knot.toml", "respect-ignore-files = false"),
-        (".ignore", "test.py"),
-        ("test.py", "~"),
-    ])?;
+    case.write_file("knot.toml", "respect-ignore-files = false")?;
```

Comment:
Ohh, was the problem that we created more and more `TestCases` in the same scope that each applied their own filters? Nice find!